aptos-move/aptos-gas-schedule/src/gas_schedule/misc.rs

```diff
@@ -144,6 +148,12 @@ impl<'a> AbstractValueSizeVisitor<'a> {
 }
 
 impl<'a> ValueVisitor for AbstractValueSizeVisitor<'a> {
+    #[inline]
+    fn visit_native_value(&mut self, _depth: usize, _id: SizedID) {
+        // TODO[agg_v2](cleanup): add a new abstract value size parameter?
+        self.size += self.params.u64;
```

Comment:
This is an abstract memory size used to track memory consumption of a Move value. It is not equal to width, e.g., `self.params.u64` is defined as 40 in gas schedule. cc @vgao1996

Because we don't have to enforce that native values have the same abstract memory size as their non-native counterparts (aggregator/snapshot structs), we can use params specific to native values which is just a u64 (roughly speaking).

Reply:
oh the 40..